src/scss/_packages.scss

```
@@ -2,80 +2,172 @@
 @import "mixins";
 
 .r-tiles {
-    @apply items-center;
-
     @screen md {
-        @apply flex;
+        @apply grid grid-cols-2 grid-flow-row-dense gap-8;
     }
 
-    .r-tile {
-        @apply rounded shadow border border-gray-300 cursor-pointer p-4 font-display;
+    .package {
 
-        width: 525px;
-        max-width: 525px;
+        .r-tile {
+            @apply rounded-xl shadow border border-gray-300 p-4 font-display;
+            @apply flex flex-col;
 
-        .logo {
-            @apply mr-4;
+            width: 100%;
 
-            width: 48px;
-            height: 48px
-        }
+            .r-tile-header {
+                @apply flex flex-row items-center mb-4;
```

`flex` is `row` by default.

```suggestion
                @apply flex items-center mb-4;
```